### PR TITLE
DSEthiop3

### PR DESCRIPTION
**Pull request by thea-m**
- I have not added the description of the script
- What would be your preferred English translation for malkəʾ?
- I cannot discern a ruling pattern for msPart p2. 
- I am not completely sure whether the measurements given on the metadata sheet for text area 2 really refer to text area 2 (for the second msPart) or to its folio size.

### CODE REVIEW

File: DayrSuryan/DSEthiop3.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>ʾArgānona Māryām</title>
```
Comment (DenisNosnitsin1970) [inline DayrSuryan/DSEthiop3.xml:8]: Don't we want to change to "Harp of Mary"?

File: DayrSuryan/DSEthiop3.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+            <title>Organ of Mary, Malkǝʾ-hymn to Mary</title>
```
Comment (DenisNosnitsin1970) [inline DayrSuryan/DSEthiop3.xml]: The simplest for Malke' is "Image of...", straight forward.

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop3.xml]: Otherwise just Malke'-hymn.

File: DayrSuryan/DSEthiop3.xml (hunk 1 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop3.xml:8]: I would be in favour of "Organ of...", which is the literatl translation and title given in the EAE. I realize that the matter of musical instruments is complicated...

File: DayrSuryan/DSEthiop3.xml (hunk 2 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop3.xml]: I used Malke'-hymn, because it is a specific genre, which this makes more obvious

**Pull request merged**